_posts/2022-08-05-hacking-for-cruise-control-ui.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title: "Hacking Strimzi for Cruise Control UI"
+date: 2022-10-31
+author: kyle_liberti
+---
+
+For some time now, Strimzi has offered Kubernetes native support Cruise Control allowing users to safely configure and communicate with Cruise Control through Strimzi custom resources.
+This is great but sometimes it is a better user experience to interact with applications like Cruise Control through a UI.
+Although Strimzi does not offer direct support for the [Cruise Control UI](https://github.com/linkedin/cruise-control-ui) it can still be run alongside and connected to a Strimzi managed Cruise Control instance.
+
+This post will walk through how to do so! 
+
+## Prerequisites
+
+For starters we will need a Strimzi managed Kafka cluster with Cruise Control enabled.  
+
+```
+                           *  *
+                        *        *
+                       *  Cruise  *
+                       *  Control *
+                        *        *
+                           *  *
+                            ^                   *  *         *  *          *  *
+                            |                *        *   *        *    *        *
++----------+                |               *  Kafka   * *  Kafka   *  *  Kafka   *
+|          |               *  *             *   Pod    * *   Pod    *  *   Pod    *
+|  Kafka   |            *        *           *        *   *        *    *        *
+| Resource | <------+  * Cluster  * ----->      *  *         *  *          *  *
+|          | +------>  * Operator *
+|          |            *        *              *  *         *  *          *  *
++----------+               *  *              *        *   *        *    *        *
+                                            * Zookeeper* * Zookeeper*  * Zookeeper*
+                                            *   Pod    * *   Pod    *  *   Pod    *
+                                             *        *   *        *    *        *
+                                                *  *         *  *          *  *
+
+Kubernetes Land                                          Kafka Cluster
```

Comment:
I am not sure I like Asciiart :-) Can we get pictures like other blog posts? Unless you want strongly show your Asciiart skills :-P

Reply:
I do love my ascii art but your approval is more important :p Updated to excalidraw at your suggestions, the diagrams still need some work but let me know if this looks better!

Reply:
Yeah they look much better :-P ... Thanks!